tsdef/twilio-video-tests.ts

```diff
@@ -103,3 +103,102 @@ function localDataTrackPublication(publication: Video.LocalDataTrackPublication)
   const publicationInfo = { kind, track };
   return publicationInfo;
 }
+
+function RemoteParticipant(RemoteParticipant: Video.RemoteParticipant) {
```

Comment:
Variable names should start with lower case, unless they are classes.
```suggestion
function RemoteParticipant(remoteParticipant: Video.RemoteParticipant) {
```